static-pages/help-center.md

```diff
@@ -46,15 +46,15 @@ title: Help Center
     <div class="card-header" id="headingOne">
       <h5 class="mb-0">
         <button class="btn btn-link" type="button" data-toggle="collapse" data-target="#collapseOne" aria-expanded="true" aria-controls="collapseOne">
-          What is Make Your Mark and what is crowdsourcing?
+          What is crowd.loc.gov and what is crowdsourcing?
         </button>
       </h5>
     </div>
 
     <div id="collapseOne" class="collapse show" aria-labelledby="headingOne" data-parent="#faqAccordion">
       <div class="card-body">
       <p>
-       Make Your Mark is an online transcription platform where anyone with an internet connection can transcribe documents from the Library of Congress’ digitized collections. We welcome anyone interested in making non-machine readable resources fully word searchable to contribute.
+       The application is an online transcription platform where anyone with an internet connection can transcribe documents from the Library of Congress’ digitized collections. We welcome anyone interested in making non-machine readable resources fully word searchable to contribute.
```

Comment:
“The application” feels a bit clunky — I'm wondering whether it'd read better as “crowd.loc.gov”